MAUI/Circular-Charts/RadialBarChart.md

```diff
@@ -0,0 +1,433 @@
+
+ * [GapRatio]() - To customize the spacing between each chart segments.
+
+ * [MaximumValue]() - To represent the span of the segment-filled area in the radial bar track. The default value of this property is `double.NaN`.
+
+ * [PaletteBrushes](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_PaletteBrushes) - To customize the appearance of the series.
+
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" />
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart type in MAUI Chart](Chart-Types_images/maui_radialbar_chart.png)
+
+## Changing the radial bar size
+
+You can use the [Radius]() property to change the radial bar chart size. The default value of the radius is [0.8]().
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        Radius = "0.5"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.Radius = 0.5;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart radius size in MAUI Chart](Chart-Types_images/maui_radius.png)
+
+## Changing the radial bar inner radius
+
+The [InnerRadius]() property of radial bar series is used to define the inner circle. The default value of this property is `0.4`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        InnerRadius = "0.2"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.InnerRadius = 0.2;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart inner radius customization in MAUI Chart](Chart-Types_images/maui_inner_radius.png)
+
```

Comment:
Change to ".NET MAUI Radial bar chart inner radius customization"

Reply:
I have changed the inner radius image alt text.